opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -354,37 +354,61 @@ def _import_id_generator(id_generator_name: str) -> IdGenerator:
     raise RuntimeError(f"{id_generator_name} is not an IdGenerator")
 
 
-def _initialize_components(auto_instrumentation_version):
-    trace_exporters, metric_exporters, log_exporters = _import_exporters(
-        _get_exporter_names("traces"),
-        _get_exporter_names("metrics"),
-        _get_exporter_names("logs"),
+def _initialize_components(
+    auto_instrumentation_version: Optional[str] = None,
+    span_exporter_names: Optional[List[str]] = None,
```

Comment:
Should we stick with `trace_exporter_names` instead?

Reply:
Fine either way. But the object seems to be called SpanExporter

Reply:
@ocelotl Any thoughts? We use both "span_exporter" and "trace_exporter" naming conventions in internal variables, but the object is called SpanExporter. I am open to either but want to make sure to get broad input on this since it is an api change that we cannot change without breaking.

Reply:
We call them "trace_exporter" because user is configuring what exporter they want under the "trace" flag. As well, the entry points are using `opentelemetry_traces_exporter`.

Reply:
Good point. Open to change it. @ocelotl @aabmass thoughts?

Reply:
As discussed in the 5/30 SIG, `traces` makes more sense because we are passing it in as a flag in `opentelemetry-instrument`, we use the term `traces_exporter` for our exporter [entrypoints](https://github.com/open-telemetry/opentelemetry-python/blob/main/exporter/opentelemetry-exporter-zipkin-json/pyproject.toml#L35) and I believe the idea was to use the signal types (traces, metrics, logs) for names.

Reply:
switched to trace_exporter_names